index.js

```diff
@@ -30,6 +30,56 @@ const getHeadSha = async (repo) => {
   return data.commit.sha;
 }
 
+async function closePRs() {
+   // Close old auto PRs
+   const {data: prs} = await octokit.pulls.list({
+     owner: 'cds-snc',
+     repo: 'notification-tf',
+     state: 'open'
+   });
+
+   prs.forEach( async pr => {
+     if(pr.title.startsWith("[AUTO-PR]")) {
+       await octokit.pulls.update({
+         owner: 'cds-snc',
+         repo: 'notification-tf',
+         pull_number: pr.number,
+         state: "closed"
+       });
+       await octokit.git.deleteRef({
+         owner: 'cds-snc',
+         repo: 'notification-tf',
+         ref: `heads/${pr.head.ref}`
+       });
+     }
+   })
+}
+
+async function hasChangesInStaging() {
+    pullsSearchDefinition = {
+        owner: 'cds-snc',
+        sort: 'updated',
+        direction: 'desc',
+        state: 'closed',
```

Comment:
Checked the documentation and discovered that it's not possible to use a `merged` state, that's why you filter on `merged_at` later. Can you add a comment in the code?